src/Lykke.Service.PaySettlement/Modules/CqrsModule.cs

```diff
@@ -0,0 +1,86 @@
+using Lykke.Cqrs.Configuration;
+using Lykke.Messaging;
+using Lykke.Messaging.RabbitMq;
+using Lykke.Messaging.Serialization;
+using Lykke.Service.PaySettlement.Cqrs;
+using Lykke.Service.PaySettlement.Settings;
+using Lykke.SettingsReader;
+using System.Collections.Generic;
+
+namespace Lykke.Service.PaySettlement.Modules
+{
+    internal class CqrsModule : Module
+    {
+        private readonly IReloadingManager<AppSettings> _appSettings;
+
+        public CqrsModule(IReloadingManager<AppSettings> appSettings)
+        {
+            _appSettings = appSettings;
+        }
+
+        protected override void Load(ContainerBuilder builder)
+        {
+            if (_appSettings.CurrentValue.PaySettlementService.ChaosKitty != null)
+            {
+                builder
+                    .RegisterType<ChaosKitty>()
+                    .WithParameter(TypedParameter.From(_appSettings.CurrentValue.PaySettlementService.ChaosKitty.StateOfChaos))
+                    .As<IChaosKitty>()
+                    .SingleInstance();
+            }
+            else
+            {
+                builder
+                    .RegisterType<SilentChaosKitty>()
+                    .As<IChaosKitty>()
+                    .SingleInstance();
+            }
+
+            builder.Register(context => new AutofacDependencyResolver(context)).As<IDependencyResolver>().SingleInstance();
+
+            var rabbitMqSettings = new RabbitMQ.Client.ConnectionFactory
+            {
+                Uri = _appSettings.CurrentValue.PaySettlementService.PaymentRequestsSubscriber.ConnectionString
+            };
+
+            builder.RegisterType<TransactionProjection>()
+                .WithParameter("multisigWalletAddress", _appSettings.CurrentValue.PaySettlementService.TransferToMarketService.MultisigWalletAddress)
+                .WithParameter("isMainNet", _appSettings.CurrentValue.PaySettlementService.IsMainNet);
+
+            builder.Register(ctx =>
+            {
+                var logFactory = ctx.Resolve<ILogFactory>();
+
+                var broker = rabbitMqSettings.Endpoint.ToString();
+
+                var messagingEngine = new MessagingEngine(logFactory,
+                    new TransportResolver(new Dictionary<string, TransportInfo>
+                    {
+                        {"RabbitMq", new TransportInfo(broker, rabbitMqSettings.UserName, rabbitMqSettings.Password, "None", "RabbitMq")}
+                    }),
+                    new RabbitMqTransportFactory(logFactory));
+
+                return new CqrsEngine(logFactory,
+                    ctx.Resolve<IDependencyResolver>(),
+                    messagingEngine,
+                    new DefaultEndpointProvider(),
+                    true,
+                    Register.DefaultEndpointResolver(new RabbitMqConventionEndpointResolver(
+                        "RabbitMq",
+                        SerializationFormat.ProtoBuf,
+                        environment: "lykke.tx-detector")),
```

Comment:
I think it should be just "lykke".

Reply:
No, it must be "lykke.tx-detector" to subscribe on 
http://rabbit-me.lykke-me.svc.cluster.local:15672/#/exchanges/%2F/lykke.tx-detector.transactions.events.exchange

Reply:
In the original code of TxDetector it is a matter of settings.